test_calculator.py

```diff
@@ -40,3 +40,21 @@ def test_div(self):
 
         result = calc.div(1., 1.e5)
         assert result == 1.e-5
+
+    def test_bitwise_not(self):
+        """Put your test cases for bitwise not that starts with assert.
```

Comment:
Please fix the description so that the purpose is easily understood at a glance.